---
layout: archive
permalink: /seminar/
author_profile: false
# redirect_from:
#  - /resume
---

{% include base_path %}

Math Seminar at CSBD
======

This is the website for the Math Seminar at CSBD. Upcoming talks and abstracts are posted here. The seminar, if not announced differently, takes place on *Thursdays at 3pm*. Currently, the organizers are [Nikola Sadovek](https://sites.google.com/view/nikola-sadovek){:target="_blank"}, Maximilian Wiesmann and [Giulio Zucal](https://sites.google.com/view/giulio-zucal/home-page){:target="_blank"}. Feel free to reach out to them if you would like to suggest a speaker or for any organizational questions.

Each talk is scheduled for 50 minutes, with an additional 10 minutes for questions.

*Location:* CSBD ([location on Google maps](https://maps.app.goo.gl/fs93agptKavkdyiG8)), top floor seminar room.


üéÑ *Between Dec 25 and Jan 08 the seminar will be on Christmas break* üéÑ


## Upcoming Talks 

| Date | Speaker | Affiliation | Title | Abstract |
| ---- | ------- | ----- | ----- | -------- | 
| Oct 30 | Lin Wan | Academy of Mathematics and Systems Science, Chinese Academy of Sciences; ELBE Visiting Faculty of CSBD | Learning Collective Multicellular Dynamics with an Interacting Mean-Field Neural SDE Model | <details> The advent of temporal single-cell RNA sequencing (scRNA-seq) data has enabled in-depth investigation of dynamic processes in heterogeneous multicellular systems. Despite remarkable advancements in computational methods for modeling cellular dynamics, integrating cell-cell interactions (CCIs) into these models remains a major challenge. This is particularly true when dealing with high-dimensional gene expression profiles from large populations of interacting cells, where the intricate interplay between cells can be obscured by data complexity. In this talk, I will present our recent work on a neural interacting mean-field stochastic differential equation (SDE) framework for temporal scRNA-seq data. Our approach combines mean-field modeling with neural networks to learn the dynamics of large, interacting cell populations directly from data. It enables the reconstruction of intrinsic cell population trajectories and the systematic characterization of CCIs. Notably, the model uncovers biologically interpretable, non-reciprocal interaction patterns and offers a principled way to study complex, non-equilibrium multicellular systems. </details> |
| Nov 06 | Iolo Jones | Durham University | tba | tba |
| Nov 13 | Thomas Bouchet | MPI CBG | An invariant theoretic approach to algebraic curves | <details> In this talk, I will present key tools from invariant theory and show how they can be used for explicit computations with algebraic curves. I will begin by introducing invariants that classify curves of a given genus up to geometric isomorphism. Beyond providing explicit equations for moduli spaces of curves, these invariants play a major role in constructing explicit examples of curves. Then, I will introduce the notion of covariants, and explain how one can reconstruct a curve/hypersurface from its invariants. I will illustrate this process through examples of curves with "interesting properties" obtained in this way. Finally, I will show how covariants can provide an efficient way to compute linear changes of variables between homogeneous polynomials, largely outperforming existing implementations. </details> |
| Nov 20 | Clemens Br√ºser | TU Dresden | tba | tba |
| Nov 27 | Daniel McGinnis | Princeton University | tba | tba |
| Dec 04 | Martin Keller-Ressel | TU Dresden | Polynomial-preserving Stochastic Processes and Applications | tba |
| Dec 11 | Sjoerd Van der Niet | Renyi Institute Budapest, TU Munich | tba | tba |
| Dec 18 | tba | tba | tba | tba |
| Jan 15 | Andreas Thom | TU Dresden | tba | tba |


## Past Talks


| Date | Speaker | Affiliation | Title | Abstract |
| ---- | ------- | ----- | ----- | -------- | 
| Oct 23 | Oskar Henriksson | MPI CBG | Exploring the parameter space of polynomial systems using pseudo-witness sets | <details> Polynomial systems that arise in applications are often *parametric* (in the sense that the coefficients depend on some model parameters), and many problems in applied math boils down to understanding what possible geometries the solution sets to such systems can take as the parameters vary. A key tool for studying such questions is the *discriminant variety*, which partitions the parameter space into regions of constant qualitative and quantitative properties of the solutions. A common challenge, however, is that many methods rely on having access to an explicit equation for the discriminant, which generally requires solving a costly elimination problem. In this talk, I will present a new approach for finding sample points in all connected components of the complement of a discriminant variety, which uses pseudo-witness sets to circumvent the need for symbolic elimination. </details> |
| Oct 16 | Renee Hoekzema | Free University of Amsterdam | Spectral gene selection methods and models for host/parasite co-phylogeny | <details>  I will talk about two disjoint projects. Firstly, I will talk about an application of spectral graph theory to the study of single cell transcriptomics, in particular the problem of feature selection of relevant genes in such experiments. Single cell transcriptomics is a powerful technique in biology that allows for the measurement of gene expression levels in many individual cells simultaneously. Current methods for analysis assume that cell types are discrete. However, in practice there is also continuous variation between cells: subtypes of subtypes, differentiation pathways, responses to environment or treatment, et cetera. We propose topologically-inspired data analysis methods that identify coherent gene expression patterns considering discrete and continuous patterns on equal footing. This is joint work with Lewis Marsh, Otto Sumray, Thomas Carroll, Xin Lu, Helen Byrne and Heather Harrington. Secondly, I will talk about ongoing work with Gillian Grindstaff on models for co-evolution of ‚Äúnested‚Äô‚Äô systems, such as parasite/host systems, individuals within a species, or ‚Äúphylosymbiosis‚Äù ‚Äì the coupled evolution of the microbiome and its hosts. We create a space of nested phylogenetic trees and study its intricate geometry. In particular we show that this space is CAT(0) ‚Äì in analogy with the influential work of Billera, Holmes and Vogtmann (2001) ‚Äì implying the existence of unique averages over nested trees. </details> |
| Oct 09 | Edmilson Roque | MPI PKS | Ergodic basis pursuit leads to robust reconstruction of sparse network dynamics | <details> Networks of coupled dynamical systems are successful models in diverse fields of science, ranging from physics to neuroscience. The network interaction structure impacts the dynamics; in fact, many malfunctions are associated with disorders in the network structure. Yet, typically, we cannot measure the interaction structure; we only have access to multivariate time series of nodes‚Äô states. This led to considerable effort in reconstructing the network from multivariate data. This reconstruction problem is ill-posed for large networks, leading to the reconstruction of false network structures. In this talk, I will present an approach that uses the network dynamics‚Äô statistical properties to ensure the exact reconstruction of weakly coupled sparse networks. Moreover, this approach exhibits robustness against noise. I will illustrate its reconstruction power using experimental multivariate time series data obtained from optoelectronic networks. </details> |
| Oct 02 | Sabina Haque | University of Michigan Ann Arbor| Graph-theoretic and algebraic geometric approaches to biochemical reaction networks | <details> Under mass-action kinetics, systems of biochemical reactions are modeled by chemical reaction networks (CRNs), a class of graphs that gives rise to polynomial dynamical systems. Approaches in this field include chemical reaction network theory and the more recent linear framework. In this talk, I will focus primarily on the linear framework, a graph-theoretic approach to timescale separation in biochemical systems. I will discuss a graph-theoretic construction within the framework that mimics what would happen if a single parameter in a graph is taken to infinity, producing what we call an asymptotic graph. I consider how properties of the asymptotic graph, such as its steady states, serve as an appropriate representation for a linear framework graph in this limit. I also speculate on some extensions of this construction beyond the scope of the linear framework to parameter identifiability and the steady state varieties of CRNs, suggesting areas for future work at the intersection of graph theory, algebraic geometry, and dynamical systems. </details> |
| Sep 25 | Cerene Rathilal | University of Kwa-Zulu Natal | On frames and the Peano compactification | <details> This talk will provide an introduction to pointfree topology and have a focus on some recent work on compactifications of frames. In [Curtis (1980): Hyperspaces of Noncompact Metric Spaces], Curtis introduced the concept of a locally non-separating remainder in order to study the hyperspace of a non-compact space $X$. Using the property of a locally non-separating remainder, Curtis established the conditions under which a Peano compactification of a connected space $X$ would exist. In this talk, we will present the analog of the concept of locally non-separating sets, in frames. We will discuss properties of sublocales, after which we define a locally non-separating sublocale and conclude by providing a generalisation for a special case of Curtis‚Äôs result. </details> |